backend/tests/Feature/ApiActions/GetHomeActionTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\ApiActions;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+/**
+ * @internal
+ *
+ * @coversNothing
+ */
+final class GetHomeActionTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testSuccess(): void
+    {
+        $user = User::factory()->create([
+            'email' => 'testGetHomeAction@example.com',
+            'password' => 'password',
+        ]);
+
+        $this->actingAs($user)
+            ->getJson('/api/test')
+            ->assertStatus(200)
+            ->assertJson([
+                'data' => [
+                    'id' => $user->id,
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property Illuminate\Database\Eloquent\Model::$id.